SkystarsH743HD target

darsanjp opened the pull request:
Added SkystarsH743HD target.

darsanjp committed:
SkystarsH743HD target

Added SkystarsH743HD target

DzikuVx commented on src/main/target/SKYSTARSH743HD/config.c:
```diff
@@ -0,0 +1,32 @@
+/*
+ * This file is part of Cleanflight.
```
Am, not, it's not Cleanflight, so please update the project name

darsanjp replied:
> Am, not, it's not Cleanflight, so please update the project name

Corrected the project name .

DzikuVx commented on src/main/target/SKYSTARSH743HD/config.c:
```diff
@@ -0,0 +1,32 @@
+{
+    pinioBoxConfigMutable()->permanentId[0] = BOX_PERMANENT_ID_USER1;
+    pinioBoxConfigMutable()->permanentId[1] = BOX_PERMANENT_ID_USER2;
+   // beeperConfigMutable()->pwmMode = true;
```
Not needed

darsanjp replied:
Removed the commented out code.

DzikuVx commented on src/main/target/SKYSTARSH743HD/target.c:
```diff
@@ -0,0 +1,60 @@
+
+    
+    //DEF_TIM(TIM4, CH3, PD14, TIM_USE_MC_MOTOR | TIM_USE_FW_SERVO, 0, 0),   // S9
+    //DEF_TIM(TIM4, CH4, PD15, TIM_USE_MC_MOTOR | TIM_USE_FW_SERVO, 0, 0),   // S10 DMA_NONE
```
Not needed, so remove

darsanjp replied:
Removed the commented out code.

DzikuVx reviewed (changes requested):
Please read https://github.com/iNavFlight/inav/blob/master/docs/policies/NEW_HARDWARE_POLICY.md

As this is a new target, this PR will not be merged if all the conditions from this document are not met

darsanjp committed:
Removed unused code.

Corrected the Project name.
Removed the commented out code.
Corrected the board name.
